src/Apps/Settings/Routes/Purchases/Components/SettingsPurchasesRow.tsx

```diff
@@ -63,13 +69,18 @@ const ORDER_COLORS = {
   CANCELED: "red100",
   FULFILLED: "green100",
   IN_TRANSIT: "black60",
+  OFFER_RECEIVED: "blue100",
   PROCESSING: "black60",
   REFUNDED: "red100",
   SUBMITTED: "black60",
   PROCESSING_APPROVAL: "black60",
   PAYMENT_FAILED: "red100",
 } as const
 
+type CommerceBuyerOrderOfferStateEnum =
+  | CommerceBuyerOfferActionEnum
+  | CommerceOrderDisplayStateEnum
```

Comment:
This seems like it will become a source of confusion if we don't resolve the tension soon, but :+1: for now